packages/terra-list/tests/wdio/multi-select-list-spec.js

```diff
@@ -0,0 +1,207 @@
+    });
+
+    Terra.should.themeCombinationOfCustomProperties({
+      testName: 'themed',
+      properties: {
+        '--terra-list-item-selected-hover-background-color': '#fdebeb',
+      },
+    });
+  });
+
+  describe('Multi Select List On Change', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/on-change-multi-select-list'));
+
+    it('should highlight and display the first item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0');
+
+    it('should highlight and display the second item', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,1');
+
+    it('should highlight and display the third item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,1,2');
+
+    it('should unhighlight and deselect the second item', () => {
+      // Shift-tab to go back, shift at end to release shift key-down
+      browser.keys(['Shift', 'Tab', 'Enter', 'Shift']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,2');
+
+    it('should unhighlight and deselect third item', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('selected item(s) 0 third item unhighlighted');
+
+    Terra.should.beAccessible();
+  });
+
+  describe('Multi Select List On Change with Max Selection', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/on-change-multi-select-list-with-max-selection'));
+
+    it('should highlight and display the first item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0');
+
+    it('should highlight and display the second item', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,1');
+
+    it('should not highlight and not display the third item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,1');
+
+    it('should unhighlight and deselect the second item', () => {
+      browser.addValue('ul li:nth-child(2)', 'Space');
+    });
+    Terra.should.matchScreenshot('selected item(s) 0 second item unhighlighted');
+
+    it('should highlight and select third item', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,2');
+
+    Terra.should.beAccessible();
+  });
+
+  describe('Multi Select List No Deselect Item', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/multi-select-list-with-no-deselect-item'));
+    Terra.should.matchScreenshot();
+
+    it('should not deselect item one', () => {
+      browser.click('ul li:nth-child(1)');
+    });
+    Terra.should.matchScreenshot();
```

Comment:
lets take add a screenshot name to both of these Terra.should.matchScreenshot(); so we can differentiate between the before and after click interaction.